sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/VMInstanceViewPatchStatusTests.cs

```diff
@@ -13,8 +13,8 @@ namespace Microsoft.Azure.Management.Compute.Tests.ScenarioTests
 {
     public class VMInstanceViewPatchStatusTests : VMTestBase
     {
-        private const string RgName = "PatchStatusRg";
-        private const string VmName = "testVm";
+        private const string RgName = "RGforSDKtestResources";
```

Comment:
Hopefully these are base names using the random name generator.  Otherwise, this is an anti-pattern - hard coding names makes rerunning tests a big problem for the next developer.  Again, FYI

Reply:
@markcowl , agreed, we were not in charge of these tests and don't currently have the knowledge to change it. We will be curating these tests in the future.